<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.652, y2:0.897727, stop:0.243781 rgba(83, 187, 255, 255), stop:1 rgba(255, 179, 249, 255))};</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 149, 245);

font: 22pt &quot;MS Sans Serif&quot;;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Health Management App</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;MS Sans Serif&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>enter medicine name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_midicine_textbar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medicine_search_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
background-color: rgb(4, 118, 159)
}

QPushButton:hover {
  background-color:rgb(0, 0, 0)
};

color:rgb(255, 255, 255);

font: 75 15pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medicine_home_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton {
background-color: rgb(4, 118, 159)
}

QPushButton:hover {
  background-color:rgb(0, 0, 0)
};

color:rgb(255, 255, 255);

font: 75 15pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>home</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>900</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>900</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>830</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_11">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>830</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_12">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>760</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_13">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>670</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_14">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>600</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_15">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>530</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_16">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>440</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_17">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>370</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_18">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>300</y>
      <width>931</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
